fv3core/utils/grid.py

```diff
@@ -645,6 +642,10 @@ def p_ref(self) -> float:
         """
         return self._vertical_data.p_ref
 
+    @p_ref.setter
```

Comment:
for my education: If we have properties that are "just" a single entity that we are getting and setting, what is the advantage of using a `@property`. I would understand that this would make sense if you could change ak and it would change p_ref accordingly. But what is the benefit of this?

Reply:
In this case, it's because I want to delete these setters later. They only exist so that I have a way to populate this data from inside the dyn_core translate class using its savepoint data.

Also, doing it this way means that `grid_data.p_ref = 1` sets `grid_data._vertical_data.p_ref` to 1. If this were a straight-up attribute, it wouldn't be possible to link with the composed instances. This isn't super relevant right now when we don't ever use those members, but down the line we may want to pass subsets of the container to functions that only use those subsets of the variables.